djangofromscratch/JWTLogin/models.py

```diff
@@ -1,11 +1,58 @@
+import re
+
 from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
+from django.core.mail import EmailMessage
+from django.db import IntegrityError
 from django.db.models import CharField, EmailField
+from django.dispatch import receiver
 from django.urls import reverse
 from django.utils.translation import gettext_lazy as _
+from django_rest_passwordreset.signals import reset_password_token_created
 
 from .managers import UserManager
 
 
+# reset_password_token_created signal triggers this fn
+@receiver(reset_password_token_created)
+def password_reset_token_created(sender, instance, reset_password_token, *args, **kwargs):
+    email_plaintext_message = "Hi user, you have initiated a reset password requests. Here's your token for it: token={}".format(
```

Comment:
Isn't it would be good if we add username/email in the message?

Reply:
Yes, We can do that